src/validators/BirthdateInputValidator.d.ts

```diff
@@ -0,0 +1,20 @@
+export declare const DATE_FORMATS: string[]
+export declare const dateMaskByFormat: {
+  'mm/dd/yyyy': (string | RegExp)[]
+  'mm/yyyy': (string | RegExp)[]
+  'mm/yy': (string | RegExp)[]
+}
+export declare const dateRegexByFormat: {
+  'mm/dd/yyyy': RegExp
+  'mm/yyyy': RegExp
+  'mm/yy': RegExp
+}
+export declare const dateStringMatchesFormat: (
+  cleansedDateString: string,
+  dateFormat: string
+) => '' | 'Please enter a valid date.'
+export declare const getMaxDateValidator: (props: any) => (value: any) => any
```

Comment:
So I've been poking around, is there a better type we could use for validators? Would something like `(value: string): bool` or some such

Reply:
I think probably the input as string would be fine but I think it also returns a string (empty is valid, otherwise it contains the error message) so likely it would  be `(value: string) => string` if we wanted to change this.